packages/tables/src/Grouping/Group.php

```diff
@@ -250,7 +250,7 @@ public function getKey(Model $record): mixed
         return Arr::get($record, $this->getColumn());
     }
 
-    public function getTitle(Model $record): string | Htmlable
+    public function getTitle(Model $record): ?(string | Htmlable)
```

Comment:
This is not valid PHP. It is very important to test your changes locally before you PR them. Have you tested that this is the only place that needs to be updated in order to support null groups?

```suggestion
    public function getTitle(Model $record): string | Htmlable | null
```

Reply:
Sorry. I did this through AI.

I have now tested it locally (with your change) and it works. It do crash on empty strings, but that's also the case in V3. I think that this behaviour (allowing null) is not really intended so I'm quite sure the tests aren't perfect for this case. Still do believe that "passing tests and not breaking stuff between V3 and V4" is better than doing nothing, even if it's worse than adding new tests and documenting this behaviour.

Reply:
We've implemented this fix for a client's ERP today and they have reported everything is back to normal. @moory-se are you able to make the suggested update? I would be glad to help if needed.

Reply:
@philmareu I changed the PR now and hope to get it accepted :)